src/components/layout/BlobBox.tsx

```diff
@@ -64,4 +31,38 @@ const BlobBox = (props: BlobBoxProps) => {
   );
 };
 
+const outputBlobs = (blobs: any) =>
+  blobs.map(
+    (blob: any, bi: number) =>
+      `
+        &.blob-${blob.id}-${bi} {
+          opacity: ${blob.opacity};
+          -moz-transform: ${`translate(${blob.translate})`} ${blob.transform};
+          -ms-transform: ${`translate(${blob.translate})`} ${blob.transform};
+          -webkit-transform: ${`translate(${blob.translate})`} ${
+        blob.transform
+      };
+          transform: ${`translate(${blob.translate})`} ${blob.transform};
+          z-index: ${blob.zIndex};
+        }
+      `
+  );
+
+const Blobs = styled.div<{ blobs: any }>`
+  display: none;
+  z-index: -1;
+  transform-style: preserve-3d;
+
+  @media (min-width: 768px) {
+    display: block;
+  }
+
+  img {
+    position: absolute;
+    overflow: visible;
```

Comment:
Why is there an overflow property on the img tag?

Reply:
Not sure. I didn't write the initial component, just moved it out of the render so the console did not spit out warnings/errors. I can look at removing it though.

Reply:
@alanhoskins we can remove it later 👍🏻